diff --git a/node_modules/blessed-contrib/index.d.ts b/node_modules/blessed-contrib/index.d.ts
index 94f6896..93a31af 100644
--- a/node_modules/blessed-contrib/index.d.ts
+++ b/node_modules/blessed-contrib/index.d.ts
@@ -336,6 +336,9 @@ declare namespace BlessedContrib {
         export class MapElement extends CanvasElement implements IHasOptions<MapOptions> {
             constructor(opts: MapOptions)
 
+            addMarker(marker: object): void;
+            clearMarkers(): void;
+
             options: MapOptions;
         }
 
@@ -431,7 +434,25 @@ declare namespace BlessedContrib {
             options: TreeOptions;
         }
 
+        export interface CarouselOptions {
+            screen: Blessed.Widgets.Screen;
+            rotate?: boolean;
+            interval?: number;
+            controlKeys?: boolean;
+        }
+
+        export class CarouselElement implements IHasOptions<CarouselOptions> {
+            constructor(opts: CarouselOptions)
 
+            move(): void;
+            next(): void;
+            prev(): void;
+            home(): void;
+            end(): void;
+            start(): void;
+
+            options: CarouselOptions;
+        }
     }
 
     export module widget {
@@ -502,6 +523,7 @@ declare namespace BlessedContrib {
     export function gaugeList(options?: Widgets.GaugeListOptions): Widgets.GaugeListElement
 
     export function donut(options?: Widgets.DonutOptions): Widgets.DonutElement
+    export function carousel(screens: ((screen: Blessed.Widgets.Screen) => void)[], options?: Widgets.CarouselOptions): void;
 
 }
 
